modules/op-prepare-opc-for-results.adoc

```diff
@@ -0,0 +1,53 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-results-for-openshift-pipelines-observability.adoc
+
+:_content-type: PROCEDURE
+[id="prepare-opc-for-results_{context}"]
+= Preparing the `opc` utility environment for querying {tekton-results}
+
+[role="_abstract"]
+Before you can query {tekton-results}, you must prepare the environment for the `opc` utility.
+
+.Prerequisites
+
+* You installed {tekton-results}.
+* You installed the `opc` utility.
+
+.Procedure
+
+. Set the `RESULTS_API` environment variable to the route to the {tekton-results} API. Run the following command:
++
+[source, terminal]
+----
+$ export RESULTS_API=$(oc get route  tekton-results-api-service  -n openshift-pipelines  --no-headers -o custom-columns=":spec.host"):443
+----
+
+. To authenticate with the {tekton-results} API, create an authentication token. Run the following command:
++
+[source, terminal]
+----
+$ oc create token sa <service-account>
+----
++
+Save the string that this command outputs.
+
+. Optional: create the `~/.config/tkn/results.yaml` for automatic authentication with the {tekton-results} API. The file must have the following contents:
++
+[source,yaml]
+----
+address: tekton-results-api-service.openshift-pipelines.svc.cluster.local # <1>
+token: <authentication-token> # <2>
+ssl:
+   roots_file_path: /home/example/cert.pem # <3>
+   server_name_override: tekton-results-api-service.tekton-pipelines.svc.cluster.local # <1>
```

Comment:
```suggestion
   server_name_override: tekton-results-api-service.<namespace-of-tekton-results>.svc.cluster.local # <5>
```
Server name override is a mechanism to set the host of the request to a known entity. For our use case, it is not going to change. And it is entirely different from the address of the apiserver.